tests/python/release_checklist/check_lerobot_v2_dataloader.py

```diff
@@ -21,18 +21,18 @@ def run(args: Namespace) -> None:
     # That means the `recording_id` needs to be set to "episode_0", otherwise the LeRobot dataloader
     # will create a new recording for episode 0, instead of merging it into the existing recording.
     # If you don't set it, you'll end up with 4 recordings, an empty one and the 3 episodes.
-    rec = rr.script_setup(args, f"{os.path.basename(__file__)}", recording_id="episode_0")
+    rec = rr.script_setup(args, f"{Path(__file__).name}", recording_id="episode_0")
```

Comment:
What's the point of the f string interpolation here? `Path.name` is already a `str`